src/FINDOLOGIC/Export/Data/Property.php

```diff
@@ -82,6 +91,8 @@ protected function setValues($values)
     {
         $this->values = [];
 
+        array_walk($values, array($this, 'checkIsAssociativeArray'));
```

Comment:
Preferably set the walk function as a closure instead of a method referenced by a string.